.htaccess

```diff
@@ -14,6 +14,19 @@
     # RewriteBase /
 
     RewriteEngine On
+
+    # Deny access
+    RewriteRule "^config(?:.default)?\.php" "-" [NC,F]
+    RewriteRule "^web\.config" "-" [NC,F]
+    RewriteRule "^(?:cli|ChangeLog|LICENSE)" "-" [NC,F]
+
+    # Skip the deny rule for js, css and png
+    RewriteCond "%{REQUEST_URI}" "^/(?:vendor|plugins)/.*\.(?:js|css|png)$"
+    RewriteRule ".?" "-" [S=1]
+
+    # Deny access to any non assets plugin files
+    RewriteRule "^(?:vendor|plugins)/" "-" [NC,F]
```

Comment:
Your are still blocking the plugins folder?

Reply:
Yes, and my curl are working correctly:

```
curl -I https://kanboard.mouvement-democratie-reelle.fr/plugins/Calendar/Assets/calendar.js
HTTP/1.1 200 OK
Set-Cookie: cluster017BAK=R896100235; path=/; expires=Mon, 15-Jan-2018 19:52:56 GMT
Date: Mon, 15 Jan 2018 18:52:38 GMT
Content-Type: application/javascript
Content-Length: 2622
Set-Cookie: cluster017=R3592744302; path=/; expires=Mon, 15-Jan-2018 19:54:50 GMT
Server: Apache
Last-Modified: Sun, 07 Jan 2018 03:05:27 GMT
Accept-Ranges: bytes
Vary: Accept-Encoding
Cache-Control: max-age=0, no-store
Expires: Mon, 15 Jan 2018 19:07:38 GMT
X-IPLB-Instance: 1049
```

```
curl -I https://kanboard.mouvement-democratie-reelle.fr/plugins/Calendar/README.md
HTTP/1.1 403 Forbidden
Set-Cookie: cluster017BAK=R896107858; path=/; expires=Mon, 15-Jan-2018 20:02:48 GMT
Date: Mon, 15 Jan 2018 18:52:33 GMT
Content-Type: text/html; charset=iso-8859-1
Set-Cookie: cluster017=R3592744302; path=/; expires=Mon, 15-Jan-2018 19:57:56 GMT
Server: Apache
X-IPLB-Instance: 17323
```

As I explained before, the curl user-agent was blocked by my hosting (OVH) and I nedded to relax the security for it to work. It always passed in my browser.

Regards.

Reply:
It seems to work if Kanboard is installed at the document root, but people could install Kanboard into a subfolder like this: `https://localhost/kanboard/`. In this scenario I get a 403.

Reply:
OK, I did not try this one…

Reply:
Are you planning to update your pull-request?